frontend/packages/ceph-storage-plugin/src/components/modals/block-pool-modal/modal-footer.tsx

```diff
@@ -63,17 +63,20 @@ export const BlockPoolModalFooter = (props: BlockPoolModalFooterProps) => {
       </Button>
       <Button
         type="button"
-        variant="primary"
+        variant={action !== 'Delete' ? 'primary' : 'danger'}
```

Comment:
Better to create an enum of all button texts.
Import and use pf `ButtonVariant` enum. 
```suggestion
        variant={action !== ButtonText.Delete ? ButtonVariant.primary' : ButtonVariant.danger'}
```